### PR TITLE
Add test for dir=rtl on mo

### PR DESCRIPTION
**Pull request by rwlbuis**

### CODE REVIEW

File: mathml/presentation-markup/operators/dir-mo-ref.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>&lt;mo&gt; RTL direction</title>
+    <meta charset="utf-8"/>
+  </head>
+  <body>
+
+    <!-- This checks prefix form in RTL mode. -->
+    _<math dir="rtl"><mo style="-webkit-margin-start: 0em; -webkit-margin-end: 0.055555555555555em;" lspace="0em" rspace="0em">+</mo><mi>x</mi></math>_
```
Comment (fred-wang) [inline mathml/presentation-markup/operators/dir-mo-ref.html]: I don't think we should use --webkit prefixes in WPT tests... Also, I think we haven't really defined https://github.com/mathml-refresh/mathml/issues/14 yet, so I'm not sure it's really interoperable test. Maybe this should use mspace instead?

**Review by fred-wang — CHANGES_REQUESTED** [review]

File: mathml/presentation-markup/operators/dir-mo-ref.html (hunk 1 above)
Comment (rwlbuis) [reply mathml/presentation-markup/operators/dir-mo-ref.html]: You are right, it did not even work in chromium/Mozilla using --webkit-margin! Using mspace fixes it.

Comment (fred-wang) [reply mathml/presentation-markup/operators/dir-mo-ref.html]: It seems we have a similar test with mo-paint-lspace-rspace.html?

**Comment by rwlbuis** [comment]
Closing, already tested by mathml/presentation-markup/operators/mo-paint-lspace-rspace.html .

**Pull request closed without merging**